tox.ini

```diff
@@ -66,6 +66,13 @@ commands =
     sphinx-build -W -b doctest -d {envtmpdir}/doctrees docs docs/_build/html
     python -m doctest README.rst
 
+[testenv:serve-docs]
+basepython = python3
+skip_install = true
+changedir = docs/_build/html
+commands =
+    python -m http.server {posargs}
+
```

Comment:
I have no opinion as to whether this is a good addition, but it's unrelated to the PR, so probably should be done separately, otherwise it's kind of sneaking in the back door.  @hynek ?

Reply:
Fair enough. I have since discovered `sphinx-autobuild`, which is a much better solution to the problem overall anyway.